content/social/2026-07-wk1-repair-in-public/carousel.html

```
@@ -0,0 +1,212 @@
+    <div class="foot"><span class="save">↓ Save this if you ship websites</span></div>
+    <div style="margin-top:18px" class="body" >Full report → <b style="color:#fff;font-weight:600">frankx.ai/research</b></div>
+  </div>
+</div>
+
+<!-- Hero (LinkedIn / X card) -->
+<div class="slide" style="width:100%;max-width:1100px;height:auto;aspect-ratio:16/9;background:var(--ink)">
+  <div class="bg hero"></div>
+  <div class="pad" style="justify-content:flex-end;padding:56px 64px">
+    <span class="kicker" style="position:absolute;top:56px;left:64px">One operator · an AI Center of Excellence</span>
+    <h1 class="h-lg" style="font-size:52px;max-width:62%">We audited our own production sites with an agent swarm.</h1>
+    <div style="margin-top:14px" class="body mute" style="max-width:60%">668 findings. 5 fix PRs. Published as-is. — frankx.ai/research</div>
```

HTMLHint (1.9.2)

[error] Duplicate of attribute name [ style ] was found.

(attr-no-duplication)